### PR TITLE
feat: create EXTRA_VOLUMES setting

### PR DESCRIPTION
**Pull request by Ian2012**
## Description

This PR allow connecting custom volumes to the jobs (in k8s and docker-compose). This is useful if someone wants to add custom files to be imported in superset as used here to run a job in superset with the dashboards created in the **tutor-contrib-oars:**  https://github.com/openedx/tutor-contrib-oars/pull/51

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @Ian2012!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: tutorsuperset/plugin.py [hunk 1]
```diff
@@ -38,6 +38,7 @@
         # Set to 0 to have no row limit.
         ("SUPERSET_ROW_LIMIT", 100_000),
         ("SUPERSET_SENTRY_DSN", ""),
```
Comment (bmtcril) [inline tutorsuperset/plugin.py:40]: Can you put a comment in here to explain the expected format?

**Review by bmtcril — COMMENTED** [review]
Just one comment, otherwise LGTM

**Comment by Ian2012** [comment]
@bmtcril what do you think?

This will probably be refactored once we have all plugin merged in tutor-contrib-oars

**Review by bmtcril — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@Ian2012 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/tutor-contrib-superset&entry.867055334=https://github.com/openedx/tutor-contrib-superset/pull/45&entry.1484655318=https://github.com/Ian2012&entry.752974735=2023-05-31+20:13&entry.1917517419=2023-06-05+19:54&entry.2133058324=Yes) so we can improve your experience in the future.

**Comment by bmtcril** [comment]
Yeah, let's take it and refactor as necessary later

**Pull request merged**